Pod/Public/TCCAnimationTileOverlayRenderer.m

```diff
@@ -92,29 +126,59 @@ -(void)drawMapRect:(MKMapRect)mapRect zoomScale:(MKZoomScale)zoomScale inContext
     TCCAnimationTileOverlay *mapOverlay = (TCCAnimationTileOverlay *)self.overlay;
     NSInteger zoomLevel = [TCCMapKitHelpers zoomLevelForZoomScale:zoomScale];
     
-    if (self.drawDebugInfo) {
-        MKTileOverlayPath path = [TCCMapKitHelpers tilePathForMapRect:mapRect zoomLevel:zoomLevel];
-        [TCCMapKitHelpers drawDebugInfoForX:path.x Y:path.y Z:path.z color:[UIColor blackColor] inRect:[self rectForMapRect:mapRect] context:context];
-    }
     
-    TCCAnimationTile *tile;
+    
+    CGRect rect = [self rectForMapRect: mapRect];
+    int tileSizeForZoomLevel = [self tccTileSizeForZoomLevel:((int)zoomLevel)];
+    int heightCount = mapRect.size.height / tileSizeForZoomLevel;
+    int widthCount = mapRect.size.width / tileSizeForZoomLevel;
+    CGFloat xDelta = rect.size.width / widthCount;
+    CGFloat yDelta = rect.size.height / heightCount;
+    CGFloat mapXDelta = mapRect.size.width / widthCount;
+    CGFloat mapYDelta = mapRect.size.height / heightCount;
```

Comment:
it looks like mapXDelata and mapYDelta will always be equal to tileSizeForZoomLevel. why compute it this way?